meta-balena-dunfell/recipes-devtools/mtools/mtools_%.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_remove = "file://initialize-direntry.patch"
```

Comment:
This needs to be removed from meta-balena-common as it won't be needed moving forward and added to the compatibility layers that require it instead.

Reply:
Cool, that commit has been cleaned up.